schema.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
     neutered boolean,
     weight_kg decimal
 );
+
+ALTER TABLE vet_clinicschema.animals ADD COLUMN species varchar;
```

Comment:
- [ ] Please add varchar limit may be(255)

Reply:
- [ ] Please add these queries:
- `UPDATE` the animal's table by setting the species column to Digimon for all animals that have a name ending in mon.
- `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
```sql
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

Reply:
> * [ ]  Please add varchar limit may be(255)

:+1: :rocket:

Reply:
> * [ ]  Please add these queries:
> * `UPDATE` the animal's table by setting the species column to Digimon for all animals that have a name ending in mon.
> * `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
> 
> ```sql
> BEGIN;
>    UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
>    UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 
> 
>    SELECT species from animals; -- verify that change was made
>    COMMIT;
>    SELECT species from animals; -- verify that change persists after commit
> ```

I have actually forgotten to put all those stuff in the sql file in the repos. :smiley: